packages/core/data-transfer/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "strict": true,
     "lib": ["ESNEXT"],
     "skipLibCheck": true,
-    "outDir": "dist"
+    "outDir": "dist",
+    "declaration": true
```

Comment:
Should we also add declarationMap?

Reply:
I don't know about this one since we probably don't want to publish the source map to the npm package. However, it makes sense in terms of debugging & contributing to the codebase.

Reply:
That shouldn't be a problem, we can add the maps to .npmignore (or beter, whitelist which files are included)

Reply:
You're right, I've added the sourceMap option & ignored the .map files with an .npmignore.